src/utils/trimSearchTerm.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * Remove '*' symbols from both sides of a string
+ *
+ * @param {string} a
+ * @return {string} Trimmed string
+ */
+export const trimSearchTerm = a => a.replace(/^(\*+)/, '').replace(/(\*+)$/, '');
```

Comment:
I would give a more meaningful name to the `a` parameter. 
```suggestion
export const trimSearchTerm = searchTerm => searchTerm.replace(/^(\*+)/, '').replace(/(\*+)$/, '');
```

Reply:
And do not forget to update jsDoc after parameter rename.